modules/persistent-storage-csi-drivers-supported.adoc

```diff
@@ -18,6 +18,7 @@ The following table describes the CSI drivers that are installed with {product-t
 
 |AWS EBS (Tech Preview) | ✅ | - | ✅
 |Google Cloud Platform (GCP) persistent disk (PD) (Tech Preview)| ✅ | - | ✅
+|Microsoft Azure Disk (Tech Preview) | ✅ | ✅ | ✅ 
```

Comment:
Do we need to remove the "Microsoft" here? @jsafrane 
Also, Azure don't support the clone feature yet.

Reply:
Including "Microsoft" here is consistent with the other entries in this table. But I am still seeing cloning as supported here. Is that correct?

Reply:
@duanwei33, upstream documents volume cloning as supported: https://github.com/kubernetes-sigs/azuredisk-csi-driver/tree/master/deploy/example/cloning
Can you please give it a quick test?

Reply:
@duanwei33 can you confirm that Azure cloning has been tested and doesn't work?

Reply:
Wei responded that we can consider Azure CSI as supporting cloning.

<br class="Apple-interchange-newline">
Wei Duan | &nbsp; | 2:34 AM (5 hours ago)
-- | -- | --
Hi Jan, As we have some discussion in slack, I think we can go with "azure clone supported" although there is AZ issue to be fixed | Hi Jan, As we have some discussion in slack, I think we can go with "azure clone supported" although there is AZ issue to be fixed